funlet-simulring/tests/funlet-simulring.test.js

```diff
@@ -60,84 +58,47 @@ const NO_FALLBACK_URL = '';
 const DEFAULT_FALLBACK_URL = NO_FALLBACK_URL;
 
 const DEFAULT_WHISPER_URL = '.?Whisper=true';
-const WHISPER_URL_WITH_CUSTOM_MESSAGE =
-  DEFAULT_WHISPER_URL + '&amp;Message=' + CUSTOM_MESSAGE_ENCODED;
+const WHISPER_URL_WITH_CUSTOM_MESSAGE = `${DEFAULT_WHISPER_URL}&amp;Message=${CUSTOM_MESSAGE_ENCODED}`;
 
 const XML_DECLARATION = '<?xml version="1.0" encoding="UTF-8"?>';
 
 const FULL_RESPONSE_SIMULRING_1_1 =
-  XML_DECLARATION +
-  '<Response>' +
-  '<Dial ' +
-  'action=".?Dial=true" ' +
-  'timeout="' +
-  DEFAULT_TIMEOUT +
-  '"' +
-  '>' +
-  '<Number url="' +
-  DEFAULT_WHISPER_URL +
-  '">' +
-  PHONE_NUMBER1 +
-  '</Number>' +
-  '<Number url="' +
-  DEFAULT_WHISPER_URL +
-  '">' +
-  PHONE_NUMBER2 +
-  '</Number>' +
-  '<Number url="' +
-  DEFAULT_WHISPER_URL +
-  '">' +
-  PHONE_NUMBER3 +
-  '</Number>' +
-  '</Dial>' +
-  '</Response>';
+  `${XML_DECLARATION}<Response>` +
```

Comment:
all these multiline strings can be written to use a single starting \` and an ending `